monitoring/command/generate.go

```diff
@@ -68,7 +68,11 @@ func Generate(cmdRoot string, sgRoot string) *cli.Command {
 				Value: "admin:admin",
 				Usage: "Credentials for the Grafana instance to reload",
 			},
-
+			&cli.StringSliceFlag{
+				Name:    "grafana.headers",
+				EnvVars: []string{"GRAFANA_HEADERS"},
+				Usage:   "Additional headers for HTTP requests to the Grafana instance",
+			},
```

Comment:
Preserving the spacing between the grafana options and the prometheus options

```suggestion
			},

```